src/common/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -31,13 +31,23 @@ ov::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
         if (input_rank.get_length() <= start_axis) {
             return false;
         }
+
+        const auto eps_d = mvn_node->get_eps();
```

Comment:
Maybe something like this? @ilya-lavrenov @t-jankowski what do you think?
```
// this function from "the art of computer programming" by Knuth 
static inline bool definitelyLessThan(double a, double b)
{
    static const double _epsilon = std::numeric_limits<double>::epsilon();
    return (b - a) > ( (std::fabs(a) < std::fabs(b) ? std::fabs(b) : std::fabs(a)) * _epsilon);
}

const auto eps_d = mvn_node->get_eps();
float eps_f;
if (definitelyLessThan(eps_d, std::numeric_limits<float>::min())) {
    // avoid 0 value
    eps_f = std::numeric_limits<float>::min();
} else {
    // we lose precision here, but I believe for this conversion this is ok
    // eps_f can take `inf` value
    // e.g. eps_d = std::numeric_limits<double>::max(); then eps_f will be inf
    eps_f  = static_cast<float>(eps_d);
}

```

Reply:
1. I already analyzed this solution by Knuth, which is older than C++11, and I find `std::nextafter()` more accurate in this context. `std::nextafter` and `std::nexttoward` provide exact closest float/double values (in compliance with standard) for comparison in our case. While Knuth's formula uses approaching calculation.
2. I think that zero value must pass on as is. It's up to model whether than usable case.
3. Above solution changes sign for eps from range [-float:min, 0)

Reply:
3. Above solution changes sign for eps from range [-float:min, 0)
I expected that eps in MVN is always > 0, but yes, I don't see any check for it in `validate_and_infer_type`  method of MVN
I believe this is an issue
___

ok, let's give it a chance ,  approve